package.json

```diff
@@ -84,18 +89,20 @@
     "hardhat": "^2.22.17",
     "hardhat-deploy": "^0.14.0",
     "hardhat-gas-reporter": "^2.2.2",
+    "jest": "^29.7.0",
     "prettier": "^3.4.2",
     "prettier-plugin-solidity": "^1.4.1",
     "solhint": "^5.0.4",
     "solidity-coverage": "^0.8.14",
+    "ts-jest": "^29.2.5",
     "ts-node": "^10.9.2",
     "typechain": "^8.3.2",
     "typescript": "^5.7.2",
     "zod": "^3.24.1"
   },
   "dependencies": {
-    "@api3/chains": "^11.8.0",
     "ethers": "^6.13.2",
+    "viem": "^2.23.2",
```

Comment:
Two big users of this package, market and auctioneer depend on this, so it's not a big deal imo